Tests/BugsnagSinkTests.m

```diff
@@ -39,7 +39,7 @@ - (void)setUp {
     config.autoNotify = NO;
     config.apiKey = @"apiKeyHere";
     config.releaseStage = @"MagicalTestingTime";
-    config.notifyURL = nil;
+    [config setEndpointsForNotify:@"http://localhost:8000" sessions:@"http://localhost:8000"];
```

Comment:
The notifyURL was cleared previous, but now this is being set to localhost:8000, is this correct? Also sessions endpoint is localhost:10000